### PR TITLE
Kubernetes 1.23.9

### PR DESCRIPTION
**Pull request by masa213f**
https://github.com/cybozu-go/neco/issues/2027

Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

File: etc/cke-template-pre.yml [hunk 1]
```diff
@@ -108,13 +109,14 @@ options:
         pluginConfig:
         - name: PodTopologySpread
           args:
-            # https://pkg.go.dev/k8s.io/kube-scheduler/config/v1beta1#PodTopologySpreadArgs
+            # https://pkg.go.dev/k8s.io/kube-scheduler/config/v1beta3#PodTopologySpreadArgs
             defaultConstraints:
             - maxSkew: 1
               topologyKey: topology.kubernetes.io/zone
               whenUnsatisfiable: ScheduleAnyway
+            defaultingType: List
```
Comment (masa213f) [inline etc/cke-template-pre.yml:117]: From KubeSchedulerConfiguration/v1beta2, we need to set `defaultingType: List` when `defaultConstraints` is not empty.
https://v1-23.docs.kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta3/#kubescheduler-config-k8s-io-v1beta3-PodTopologySpreadArgs
> DefaultConstraints defines topology spread constraints to be applied to Pods that don't define any in pod.spec.topologySpreadConstraints. .defaultConstraints[*].labelSelectors must be empty, as they are deduced from the Pod's membership to Services, ReplicationControllers, ReplicaSets or StatefulSets. When not empty, .defaultingType must be "List".

**Review by morimoto-cybozu — COMMENTED** [review]
LGTM

**Review by kmdkuk — APPROVED** [review]
LGTM

**Comment by masa213f** [comment]
Rebased. Makefile.common is conflicted.

```
diff --cc Makefile.common
index 903af47c,f0b84fdd..00000000
--- a/Makefile.common
+++ b/Makefile.common
@@@ -1,12 -1,12 +1,21 @@@
  # tool versions used for both local bin and deb/zip packages
  
  ## These should be updated when Kubernetes is updated
++<<<<<<< HEAD
 +CKE_VERSION = 1.22.9 # CKE version is 
 +CONTAINERD_VERSION = 1.5.8
 +NERDCTL_VERSION = 0.20.0
 +CRITOOLS_VERSION = 1.22.0
 +RUNC_VERSION = 1.0.3
 +K8S_VERSION = 1.22.12
++=======
+ CKE_VERSION = 1.23.0-rc.2
+ CONTAINERD_VERSION = 1.6.6
+ NERDCTL_VERSION = 0.22.2
+ CRITOOLS_VERSION = 1.24.2
+ RUNC_VERSION = 1.1.3
+ K8S_VERSION = 1.23.9
++>>>>>>> d10b5314... Kubernetes 1.23.9
  STERN_VERSION = 1.21.0
  ```

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**